docs/reference/tctl/workflow.md

```diff
@@ -0,0 +1,16 @@
+---
+id: workflow
+title: tctl workflow
+description: How to operate Workflow Executions using tctl.
+tags:
+  - reference
+  - tctl
+---
+
+<!-- prettier-ignore -->
+import * as WhatIsAWorkflowExecution from '../../content/what-is-a-workflow-execution.md'
+import * as TctlWorkflowShow from './workflow/show.md'
+
+The `tctl workflow` commands operate <preview page={WhatIsAWorkflowExecution}>Workflow Executions</preview>.
+
+- <preview page={TctlWorkflowShow}>show</preview>
```

Comment:
I was thinking that this page would be the roll up -

It would include all of the subcommands for this command what do you think?

Reply:
That's the plan. I haven't yet updated the metadata in the other pages to work nicely with the preview.

Reply:
I added list items for `showid` and `start`. Something is amiss in our rendering for lists, methinks.

Reply:
This page no longer uses an unordered list, but the rendering issue remains. I'll file a GitHub issue.